plane_spotter/geolocator.py

```diff
@@ -1,15 +1,87 @@
 from typing import Any
 
+import csv
+import haversine
+
+AIRPORT_TYPE_BLACKLIST = set(('balloonport', 'closed', 'heliport', 'seaplane_base'))
+
 class Geolocator:
   ''' Translates coordinates to the closest airport. '''
 
   def __init__(self, airport_code_file: str='plane_spotter/data/airport-codes.csv') -> 'Geolocator':
     ''' Initializes a Geolocator object '''
-    pass
+    self.__airports = []
+    # Add airport dict for each row in airport_code_file
+    with open(airport_code_file, 'r') as file:
+      reader = csv.reader(file)
+      headers = next(reader)
+      for line in reader:
+        airport = {}
+        # Convert row to dict
+        for key, value in zip(headers, line):
```

Comment:
You can use a `DictReader` to get the key-val pair for each row with headers already in it.

https://docs.python.org/3/library/csv.html#csv.DictReader

Reply:
Love it.